test/odbc/psqlodbc/psqlodbc_tests_common.h

```diff
@@ -412,6 +412,10 @@ void testComparisonFunctions(ServerType serverType, const string &tableName, int
  * @param expectedResults 2D vector containing the expected results for each operation.
  * @param expectedLen A vector containing the expected length of all results from each operation. 
 */
+
+void testStringFunctions(ServerType serverType, const string &tableName, const vector<string> &operationsQuery, 
+vector<vector<string>> &expectedResults, const vector<string> &pk, const string &Col_NAME);
```

Comment:
Col_NAME is not a good name. Commit to either pascal, camel, or snake case please. IN this case, you did camel so let's just rename this colName